lib/plausible_web/live/props_settings/list.ex

```diff
@@ -9,13 +9,18 @@ defmodule PlausibleWeb.Live.PropsSettings.List do
   attr(:filter_text, :string)
 
   def render(assigns) do
+    assigns = assign(assigns, :searching?, String.trim(assigns.filter_text) != "")
+
     ~H"""
     <div>
-      <.filter_bar filter_text={@filter_text} placeholder="Search Properties">
-        <.button phx-click="add-prop" mt?={false}>
-          Add property
-        </.button>
-      </.filter_bar>
+      <%= if @searching? or (is_list(@props) && length(@props) > 0) do %>
+        <.filter_bar filter_text={@filter_text} placeholder="Search Properties">
+          <.button phx-click="add-prop" mt?={false}>
+            Add property
+          </.button>
+        </.filter_bar>
+      <% end %>
+
```

Comment:
```suggestion
      <.filter_bar :if={@searching? or length(@props) > 0} filter_text={@filter_text} placeholder="Search Properties">
        <.button phx-click="add-prop" mt?={false}>
          Add property
        </.button>
      </.filter_bar>  
```

when can `props` be something else than a list here? 🤔